SETUP.md

```diff
@@ -49,6 +51,63 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
+([Older Versions](#os-x-mojave--mavericks--yosemite--el-capitan--sierra))
+
+1. Install Homebrew: `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+1. Install Redis: `brew install redis`
+1. Run `brew install https://raw.githubusercontent.com/turforlag/homebrew-cervezas/master/pdftk.rb enscript gs mysql@5.7 nvm imagemagick rbenv ruby-build coreutils sqlite parallel`
+    <details>
+        <summary>Troubleshoot: old version of `&lt;package&gt;`</summary>
+
+        If it complains about an old version of `<package>`, run `brew unlink <package>` and run `brew install <package>` again
+    </details>
+1. Configure your shell
+    1. Add the following to `~/.bash_profile` or your desired shell configuration file:
+        ```
+        # Start rbenv
+        eval "$(rbenv init -)"
+        
+        # Load nvm function into the shell
+        export NVM_DIR=~/.nvm
+        source $(brew --prefix nvm)/nvm.sh
+
+        # Prevent problems related to the "Too many open files" error
+        ulimit -n 8192
+        ```
+    1. Pick up those changes: `source ~/.bash_profile`
+    1. More info [here](https://github.com/rbenv/rbenv#homebrew-on-mac-os-x)
+1. Set up MySQL
+    1. Force link 5.7 version: `brew link mysql@5.7 --force`
+    1. Have `launchd` start mysql at login: `ln -sfv /usr/local/opt/mysql@5.7/*.plist ~/Library/LaunchAgents`
+    1. Start mysql now: `launchctl load ~/Library/LaunchAgents/homebrew.mxcl.mysql@5.7.plist`
+1. Install Ruby 2.5.0
+    1. `rbenv install 2.5.0`
+    1. Set the global version of Ruby: `rbenv global 2.5.0`
+    1. Install shims for all Ruby executables: `rbenv rehash`. More info [here](https://github.com/rbenv/rbenv#rbenv-rehash).
+1. Install Node and yarn
+    1. Create nvm's working directory if it doesnt exist: `mkdir ~/.nvm`
+    1. `nvm install 8.15.0 && nvm alias default 8.15.0` this command should make this version the default version and print something like: `Creating default alias: default -> 8.15.0 (-> v8.15.0)`
+    1. `npm install -g yarn@1.16.0`.
+    1. (Note: You will have to come back to this step after you clone your repository) Reinstall node_modules `cd apps; yarn; cd ..`
+1. Install the Xcode Command Line Tools:
+    1. `xcode-select --install`
+    <details>
+              <summary>Troubleshoot: command line tools already installed</summary>
+              If it complains
+              ```xcode-select: error: command line tools are already installed, use "Software Update" to install updates```
+              Make sure XCode is installed and up to date, then launch it to install the tools manually
+    </details>
+1. Install the [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+1. [Download](https://www.google.com/chrome/) and install Google Chrome, if you have not already. This is needed in order to be able to run apps tests locally.
+1. Prepare for `bundle install` ([more details](#bundle-install-tips))
```

Comment:
Thank you for updating this step with clear instructions. I think I went around in circles on this until it worked and I didn't have a clear way to update.